src/values.ts

```diff
@@ -0,0 +1,17 @@
+import { purry } from './purry';
+
+function _values<T>(object: { [key: string]: T } | ArrayLike<T>) {
+  return Object.values(object) as T[];
+}
+
+// data-first
+export function values<T>(object: { [key: string]: T } | ArrayLike<T>): T[];
+
+// data-last
+export function values<T>(): (
+  object: { [key: string]: T } | ArrayLike<T>
+) => T[];
```

Comment:
I don't really think we need a data last version of this function. There are also no tests for it. I tried it out with `pipe({ 1: 'one', 2: 'two', 3: 'three' }, values)` and the inferred type is not correct. It looks good when removing the data last overload, and since we also don't have tests for it, I'd suggest to remove it.

Reply:
My initial intention of use was for data last, since there is some piping that needs it.

I can further work more on the testing and getting everything worked with data last if it's ok.

Reply:
With only one argument list, it’s implicitly data last. See the previous example, which works:

`pipe({ 1: 'one', 2: 'two', 3: 'three' }, values)`

with an explicit data last version, I think we'd need to do:

`pipe({ 1: 'one', 2: 'two', 3: 'three' }, values())`

And I don’t see the advantage in that

Reply:
Oh I see. I didn't quite understand that. Neat. I'll finish it up then 😄